user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Item.md

```diff
@@ -0,0 +1,166 @@
+Copy the below snippet and change the content as you see fit using your favorite editor.
+
+```md
+---
+uid: Connector_Example_Rates_Custom
+---
+
+# Example_Rates_Custom
+
+This is a connector that serves as example on how to integrate bit rates.
+
+## About
+
+
+This catalog item has been made available as part of the Catalog Registration [Tutorial](https://docs.dataminer.services/tutorials/Tutorials.html)
+```
+
+* a manifest.yml file describing the properties of the catalog item.  
+Copy the below snippet and change the fields as you see fit using your favorite editor.
+
+> [!Note]  
+Make sure to obtain a new Guid to uniquely store and identify the catalog item.  
+Navigate to [guidgenerator](https://guidgenerator.com/) and create a new one using format *hyphens* or use your prefered method.
+
+```yml
+# [Required]
+# Possible values for the catalog item that can be deployed on a DataMiner System:
+#   - automationscript: If the catalog item is a general-purpose DataMiner Automation script.
+#   - lifecycleserviceorchestration: If the catalog item is a DataMiner Automation script designed to manage the life cycle of a service.
+#   - profileloadscript: If the catalog item is a DataMiner Automation script designed to load a standard DataMiner profile.
+#   - userdefinedapi: If the catalog item is a DataMiner Automation script designed as a user-defined API.
+#   - adhocdatasource: If the catalog item is a DataMiner Automation script designed for a ad hoc data source integration.
+#   - chatopsextension: If the catalog item is a DataMiner Automation script designed as ChatOps Extension.
+#   - connector: If the catalog item is a DataMiner XML connector.
+#   - slamodel: If the catalog item is a DataMiner XML connector designed as DataMiner Service Level Agreement model.
+#   - enhancedservicemodel: If the catalog item is a DataMiner XML connector designed as DataMiner enhanced service model.
+#   - visio: If the catalog item is a Microsoft Visio design.
+#   - solution: If the catalog item is a DataMiner Solution.
+#   - testingsolution: If the catalog item is a DataMiner Solution designed for automated testing and validation.
+#   - samplesolution: If the catalog item is a DataMiner Solution used for training and education.
+#   - standardsolution: If the catalog item is a DataMiner Solution which is a out-of-the-box solution for specific use case or application.
+#   - dashboard: If the catalog item is a DataMiner Dashboard.
+#   - lowcodeapp: If the catalog item is a DataMiner low-code app.
+#   - datatransformer: If the catalog item is a Data Transformer.
+#   - dataquery: If the catalog item is a GQI data query.
+#   - functiondefinition: If the catalog item is a DataMiner function definition.
+#   - scriptedconnector: If the catalog item is a DataMiner scripted connector.
+#   - bestpracticesanalyzer: If the catalog item is a DataMiner Best practices Analysis file.
```

Comment:
Do we want to keep all the possible types + explanation here?
If something changes we'll need to remember to update it here as well

Reply:
its good to have a template they can just copy to get started, i dont expect it to change that often that it will give issues. we might think if this can be shared in another way though, maybe through git. we have an explanation in another location about the types, but the syntax/formatting of the yaml is not defined there.